todo/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { markDone, addTodo } from './actions';
+
+const style = { textDecoration: 'line-through' };
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userInput: ''
+    }
+  }
+
+  handleSubmit = event => {
+    this.setState({
+      userInput: event.target.value
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <ul>
+          {this.props.todos.map((todo) => (
+            <li
+              style={todo.completed ? style : null} 
```

Comment:
Nice use of a ternary operator here. I also like how you broke out the styling into its own variable up on line # 5 👍